### PR TITLE
RCAL-289 Updates for DMS release 0.5

### LINKED ISSUE #441: Update documentation for build 0.5
_Issue [RCAL-289](https://jira.stsci.edu/browse/RCAL-289) was created on JIRA by [Nadia Dencheva](https://jira.stsci.edu/secure/ViewProfile.jspa?name=dencheva):_

### PR DESCRIPTION
**Pull request by ddavis-stsci**
GitHub issue, Closes #441 

Resolves [RCAL-289](https://jira.stsci.edu/browse/RCAL-289)

**Description**

This PR updates the documentation for DMS build 0.5

Checklist
- [ ] Tests, N/A

- [x] Documentation

- [x] Change log

- [x] Milestone

- [x] Label(s)

### CODE REVIEW

File: docs/roman/pipeline/exposure_pipeline.rst [hunk 1]
```diff
@@ -73,12 +74,15 @@ for errors and Data Quality flags created and initialized to zero.
...
+:Data model: `~romancal.datamodels.ImageModel`
+:File suffix: _cal
 
-Result of applying all pipeline steps up through the :ref:`jump <jump_step>` step,
-to produce corrected and CR-flagged 2D ramp data, which will have one less data dimensions
-as the input raw 3D data (ncols x nrows x ngroups ).
+Result of applying all pipeline steps up through the
+:ref:`flatfield <flatfield_step>` step, to produce corrected flatfield data
+which is 2D image data, which will have one less data dimensions as the input
+raw 3D data (ncols x nrows x ngroups). In addition to being a 2-dimensional
```
Comment (PaulHuwe) [inline docs/roman/pipeline/exposure_pipeline.rst:86]: The order of dimensions should be: ngroups x ncols x nrows

File: docs/roman/ramp_fitting/description.rst [hunk 2]
```diff
@@ -13,17 +13,21 @@ more detail below.
...
 Segments are determined using
 the 3-D GROUPDQ array of the input data set, under the assumption that the jump
 step will have already flagged CR's. Segments are terminated where
-saturation flags are found. Pixels are processed simultaneously in blocks 
+saturation flags are found. Pixels are processed simultaneously in blocks
 using the array-based functionality of numpy.  The size of the block depends
 on the image size and the number of groups.
 
+The ramp fitting step is also where the :ref:`reference pixels <refpix>` are
+trimmed resulting in a smaller array being passed to the subsequent steps.
```
Comment (PaulHuwe) [inline docs/roman/ramp_fitting/description.rst:26]: suggested  "-" or "," between trimmed and resulting.

**Review by PaulHuwe — CHANGES_REQUESTED** [review]

File: docs/roman/pipeline/exposure_pipeline.rst [hunk 3]
```diff
@@ -40,9 +41,9 @@ The ``exposure`` pipeline has one optional argument::
   --save_calibrated_ramp  boolean  default=False
 
 If set to ``True``, the pipeline will save intermediate data to a file as it
-exists at the end of the :ref:`jump <jump_step>` step (just before ramp fitting). The data
-at this stage of the pipeline are still in the form of the original 3D ramps
-(ncols x nrows x ngroups) and have had all of the detector-level
+exists at the end of the :ref:`jump <jump_step>` step (just before ramp fitting).
+The data at this stage of the pipeline are still in the form of the original
+3D ramps (ncols x nrows x ngroups) and have had all of the detector-level
```
[the remark this answers is not in the document]
Comment (ddavis-stsci) [reply docs/roman/pipeline/exposure_pipeline.rst]: done

File: docs/roman/pipeline/exposure_pipeline.rst (hunk 1 above)
Comment (ddavis-stsci) [reply docs/roman/pipeline/exposure_pipeline.rst]: done

File: docs/roman/ramp_fitting/description.rst (hunk 2 above)
Comment (ddavis-stsci) [reply docs/roman/ramp_fitting/description.rst]: added

**Review by PaulHuwe — APPROVED** [review]
LGTM

**Pull request merged**